### PR TITLE
[.NET] Note obsolete API removed

### PR DESCRIPTION
**Pull request by bruno-garcia**
/cc @Tyrrrz

### CODE REVIEW

**Comment by Tyrrrz** [comment]
@bruno-garcia I'm not aware of any other such APIs, to be honest.

File: src/platforms/dotnet/migration.mdx [hunk 1]
```diff
@@ -50,6 +50,11 @@ The motivation was to remove the reference to `System.Web` to decouple the core
 
 To match Sentry's default environment naming convention, unless explicitly set otherwise, the Sentry environment will be reported lowercase. That means if `ASPNET_ENVIRONMENT` is equal to `Production`, the Sentry SDK will report it as `production`.
 
+## ASP.NET Core `IncludeRequestPayload`
+
+The option `IncludeRequestPayload` that was deprecated in the Sentry SDK 2.0 was finally removed.
+The replacement has been the option `MaxRequestBodySize`.
```
Comment (imatwawana) [inline src/platforms/dotnet/migration.mdx:56]: Suggest this change to wording to make this content a bit more evergreen. 

```suggestion
The option `IncludeRequestPayload` that was deprecated in the Sentry SDK 2.0 has been removed.
The replacement is the option `MaxRequestBodySize`.
```

**Review by imatwawana — APPROVED** [review]
Minor wording suggestions

**Comment by bruno-garcia** [comment]
Thanks @imatwawana

**Pull request merged**